### PR TITLE
Updated Gradle build files to only deploy JAR artifacts for packages wit...

### PR DESCRIPTION
**Pull request by peci1**
...h sources present in the local workspaces.

Otherwise, nonsense JARs with just the manifest and version numbers taken from rosjava_messages' last tag are generated.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -61,5 +66,3 @@ task distclean << {
         }
     }
 }
-
-defaultTasks 'publishMavenJavaPublicationToMavenRepository'
```
Comment (stonier) [inline build.gradle:65]: Any particular reason why the defaultTasks is dropped?

**Comment by stonier** [comment]
Good pickup. Just the one question above.

**Comment by peci1** [comment]
It is moved to build.gradle.template to only be executed if the NPE is not thrown.

File: build.gradle (hunk 1 above)
Comment (stonier) [inline build.gradle:65]: Ah missed it - it is moved to the build.gradle.template below.

**Comment by stonier** [comment]
ok, looks great. Will have to wait/test it on my machine at work. Only have trusty/indigo here and that will be using something different.

**Pull request merged**